logging/google/cloud/logging/handlers/__init__.py

```diff
@@ -19,3 +19,6 @@
     ContainerEngineHandler)
 from google.cloud.logging.handlers.handlers import CloudLoggingHandler
 from google.cloud.logging.handlers.handlers import setup_logging
+
+__all__ = ['AppEngineHandler', 'CloudLoggingHandler', 'ContainerEngineHandler',
```

Comment:
ISTM we should be disabling that check for package-level exports, rather than adding non-DRY `_all_` which we would otherwise avoid.